packages/components/src/date-time/date/styles.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../button';
+import { COLORS } from '../../utils';
+
+// Styles that overrides the calendar styling provided by react-dates go in
+// style.scss. Everything else goes here.
+
+export const Day = styled.div< { hasEvents: boolean } >`
+	align-content: center;
+	display: flex;
```

Comment:
It did not occur to me! I've swapped it out for `VStack`. This stuff is pretty new to me... do you think there's a benefit one way or the other? On the one hand it's nice that by using `display: flex` all the styling is together in `styles.ts`, but on the other hand it's nice that by using `VStack` we can remove a few lines of code.

Reply:
The reason why I'm suggesting `VStack` is mostly because I believe that by being the first users of our own components, we can have more chances at improving the library and at promoting components composition